fvm/meter/computation_meter.go

```diff
@@ -96,6 +96,23 @@ func (m *ComputationMeter) MeterComputation(
 	return nil
 }
 
+// HasComputationCapacity returns true if enough computation capacity has left in the transaction
+func (m *ComputationMeter) HasComputationCapacity(
+	kind common.ComputationKind,
+	intensity uint,
+) bool {
+	w, ok := m.params.computationWeights[kind]
+	// if not found use a default weight
+	if !ok {
+		w = 1 << MeterExecutionInternalPrecisionBytes
```

Comment:
default weights are 0, so if the weight is not found there is always enough computation.

```go
if !ok { return true }
```

Reply:
Updated that.